release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2784,3 +2784,30 @@ link:https://access.redhat.com/solutions/6258591[{product-title} 4.7.24 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-7-28"]
+=== RHSA-2021:3262 - {product-title} 4.7.28 bug fix and security update
+
+Issued: 2021-09-01
+
+{product-title} release 4.7.28 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2021:3262[RHSA-2021:3262] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:3263[RHSA-2021:3263] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6284521[{product-title} 4.7.28 container image list]
+
+[id="ocp-4-7-28-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Insights widget did not display to the user that 0 issues were found when selecting `Insights status` for a cluster. Consequently, an empty widget with no information was presented to users. This update adds additional information to cases that return 0 issues when selecting *Insights status* for a cluster. As a result, when a cluster is healthy, the widget shows a chart with `Total Issues = 0` and provides a link to the OpenShift Cluster Manager. (link:https:/bugzilla.redhat.com/show_bug.cgi?id=1986724[*BZ#1986724*])
```

Comment:
There's a mismatch between the text formatting styles of \`Insights status\` and \*Insights status\*. I would opt for the bolded version (*) since you are referring to it being selectable.

Reply:
Done! Thanks for the suggestion.